operator/src/main/java/org/bf2/operator/managers/IngressControllerManager.java

```diff
@@ -625,14 +617,20 @@ static LongSummaryStatistics summarize(List<Kafka> kafkas, Function<Kafka, Strin
 
     int numReplicasForDefault(long connectionDemand) {
         // use the override if present
+        int minimumReplicaCount = nodeInformer.getList().size() > 0 ? 1:0;
         if (defaultReplicaCount.isPresent()) {
             return defaultReplicaCount.get();
+        } else if (nodeInformer.getList().size() > 3){
+            // enforce a minimum of two replicas on clusters that can accommodate it when no default specified
+            // which may change if we don't want to
+            // provide pod / node level HA for the az specific replicas.
```

Comment:
This method is specifically for the bootstrap / default, which we still want to be HA.